monai/visualize/__init__.py

```diff
@@ -9,8 +9,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .visualizer import default_normalizer, default_upsampler  # isort:skip
-from .class_activation_maps import CAM, GradCAM, GradCAMpp, ModelWithHooks
+from .visualizer import default_upsampler  # isort:skip
```

Comment:
perhaps remove the isort:skip?

Reply:
The skip is needed unfortunately. Otherwise `visualizer` will be moved to the end (alphabetically) and then there's a circular dependency.